datairisplatform/malloy-py
# [malloy] make a python snowflake connector in malloy

## Summary

- implement snowflake connector
- add a few tests to verify the connector can run sql
- ensure it can also be used with malloy runtime
- add some runtime tests (which should work locally)

## Test plan
Know about # need to also follow https://medium.com/@alexishevia/the-magic-behind-npm-link-d94dcb3a81af
we must properly link malloy code that has snowflake support (locally vs since downloaded prod npm packages will not have this)


```
cd submodules/malloy-service
npm ci && npm run malloy-link && npm run build && npm run start

PYTHONPATH=src python -m pytest tests/malloy/data/snowflake/test_snowflake_connection.py gst

PYTHONPATH=src python -m pytest tests/malloy/test_runtime.py
```
